site/guide/swap-documentation-templates.qmd

```diff
@@ -64,31 +65,38 @@ When swapping templates, only the document structure is changed. Any modificatio
 
 7. On the right, select a different template or version:
    
-   - **Template**: Change to a different template entirely
-   - **Version**: Change to a different version of the template you selected
+   - **Template** — Change to a different template entirely
+   - **Version** — Change to a different version of the template you selected
    
-   For example: Select a previous version of the template currently in use to revert to that version. 
+   For example: Select a newer version of the template currently in use to switch to that version. 
    
    After you select a different template or version, the YAML differences between the templates are highlighted. 
 
 8. Click **Prepare Swap**.
 
 9. Enter a note to enable completing the swap and click **Swap Template**.
 
-After your model documentation template has been swapped successfully, you can continue to work on your model documentation
+::: {.column-margin}
+^2^ [Customize documentation templates](customize-documentation-templates.qmd)
+:::
 
-#### Reusing old content
+![](swap-template-version.gif){fig-alt="A gif showcasing swapping a documentation template version."}
```

Comment:
I know we already have the workflows docs where the animated GIF appears in a similar location after the steps, but it feels like an animated GIF that covers ALL THE STEPS described in this section should perhaps live BEFORE the steps? That's where we'd place a 'show me how' video if we had one, I think. 

Maybe something to chat about by way of establishing a convention.

Reply:
Hm, good point. I sort of just placed these based on some existing docs (that I didn't write). But my personal preference is after, but I'm not a videos person is maybe why — I tend to want to read the instructions myself first, then look for clarification via different methods after if the wording is unclear. Generally screencaps do go AFTER the written word though. 🤔 (And it doesn't always capture the ENTIRE flow, usually just the END of the flow in the case of these templates, the bit with the big menu that comes up that might be confusing.)